src/animators/easingAnimator.ts

```diff
@@ -72,7 +72,7 @@ module Plottable.Animators {
      * @param {number} startDelay The start delay in milliseconds.
      * @returns {Easing} The calling Easing Animator.
      */
-    public startDelay(startDelay: number): Easing;
+    public startDelay(startDelay: number): this;
     public startDelay(startDelay?: number): any {
```

Comment:
why are so many method return types annotated as `any`? shouldn't it be `Easing | number` here (or `this | number` with TS 1.7.3)? was this code never updated to use union types?